### PR TITLE
ActivityLog: Filter bar visual tweaks and fixed tablet layout

### PR DESCRIPTION
**Pull request by Initsogar**
Solves https://github.com/Automattic/jetpack-backup-team/issues/259 and https://github.com/Automattic/jetpack-backup-team/issues/260

## Proposed Changes

* Fix layout weirdness in tablets
* Fix Activity Type vertical overflow
* Remove ellipsis in the clear buttons in some resolutions
* Clear buttons now have a border-left to avoid having two icons in close proximity

CleanShot 2023-11-29 at 21 17 11

## Testing Instructions

* Spin up a Calypso and Jetpack Cloud live branch
* Navigate to the Activity Log.
* Play around with the filters and ensure it is not breaking UI when date range and activity types.
* To test the changes, I recommend you to play around with different resolutions (using Chrome Developer Tools).

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/my-sites/activity/filterbar/style.scss [hunk 1]
```diff
@@ -392,7 +393,7 @@
 			width: 24px;
 		}
 
-		@include break-xlarge {
+		@include breakpoint-deprecated( ">660px" ) {
```
Comment (Initsogar) [inline client/my-sites/activity/filterbar/style.scss:396]: Using this deprecated breakpoint to ensure everything tweaks at the same viewport.

**Review by keoshi — APPROVED** [review]
Changes look great, @Initsogar. Thanks again for refining the UI so quickly.

Only noticed two things we could tweak, but they are non blockers:

1. In some resolutions, the entire search/filtering UI gets pushed to the right by 16px. It seems to come from `.filterbar` styles.

image


2. The left border on the clear button seems a bit harsh. Changing it from `--color-text-inverted` to `--color-border` leads to a more natural look. Here's a quick before/after

Before | After
--|--
image | image

**Comment by Initsogar** [comment]
Thanks @keoshi! Good catches there.

I just pushed the two fixes:
* Removed the 16px left/right margin on the filterbar
* Switched `--color-text-inverted` to `--color-border`

**Comment by keoshi** [comment]
@Initsogar The border looks great and I don't see the extra spacing anymore. However, I think we need to keep the `16px` side margin on screens smaller than 600px; see this:

image

**Comment by Initsogar** [comment]
> @Initsogar The border looks great and I don't see the extra spacing anymore. However, I think we need to keep the `16px` side margin on screens smaller than 600px

My eyes tricked me when I tested it. You are right. Just added the 660px breakpoint (the same we use for switching the filterbar layout into mobile). Fixed!

**Pull request merged**